packages/idos-sdk-js/README.md

```diff
@@ -419,6 +419,37 @@ const signer = selector.wallet();
 const address = (await signer.getAccounts())[0].accountId
 ```
 
+### Inserting a DAG
```

Comment:
This should be in the AG section, and se should point it out that this flow is for passporting.

Reply:
Found myself explaining passporting too much. Going to just remove this section for now.